src/use_case/generate_meal_by_id/GenerateMealByIDOutputData.java

```diff
@@ -0,0 +1,109 @@
+package use_case.generate_meal_by_id;
+
+import use_case.generate_meal.GenerateMealOutputData;
```

Comment:
Watch out for this. When you copy and paste from other classes Intellij refactors and assume you want the class there. Also why is `GenerateMealOutputData` here?